README.md

```diff
@@ -6,6 +6,82 @@ multibot
 
 A programmatic multi-repository helper.
 
+Multibot allows you to invoke actions (`read`, etc.) on 1+ files in 1+
+repositories concurrently. It is meant to be a helper tool for multi-project
+updates, branches, pull requests, etc.
+
+## Install
+
+```sh
+$ npm install multibot
+```
+
+## Usage
+
+```
+A friendly multi-repository robot.
+
+Usage: multibot --action=<string> [options]
+
+Options:
+  --action       Actions to take                               [string] [required] [choices: "read"]
```

Comment:
Read could be default, but I like having it be specific. I would have done it as a `multibot <action>` instead of a flag but yargs totally could not work with that as it's supposed to be documented to do.

Offhand, future actions are:

* `--action=commit` -- Add a commit of a transform to a branch.
* `--action=branch` -- Create a new branch.
* ... maybe some aggregation of those two above...